src/Runner.Worker/action_yaml.json

```diff
@@ -7,7 +9,8 @@
                     "name": "string",
                     "description": "string",
                     "inputs": "inputs",
-                    "runs": "runs"
+                    "runs": "runs",
+                    "outputs": "composite-outputs"
```

Comment:
is `composite-outputs` or just `outputs` even it only matter for composite actions for now?